### PR TITLE
*: cut v0.5.0

### PR DESCRIPTION
**Pull request by brancz**
It's been almost 3 months since the last release and we have a bunch of new features that were added since the last release, so we should do another release.

@fabxc @mxinden @andyxning 

/cc @piosz (to push the new image once tagged)

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/132)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## v0.5.0 / 2017-05-03
+
+* [FEATURE] Add label metrics for Pods, Nodes and Services.
+* [FEATURE] Expose number number of ready Pods for DaemonSets.
```
Comment (andyxning) [inline CHANGELOG.md]: nit
* duplicate number. :)

Comment (brancz) [inline CHANGELOG.md]: nice catch! fixed :slightly_smiling_face:

**Comment by fabxc** [comment]
:+1:

**Pull request merged**